colour/characterisation/aces_it.py

```diff
@@ -1124,7 +1121,8 @@ def matrix_idt(
         XYZ_to_optimization_colour_model,
         finaliser_function,
     ) = optimisation_factory()
-    optimisation_settings = {
+
+    optimisation_settings: dict[str, Any] = {
```

Comment:
without this, it'd be inferred as `dict[str, str]`, which due to its invariant type-parameters, would lead to it being rejected when passed as `**kwargs` to `minimize` below